sdk/types/src/build.ts

```diff
@@ -2,12 +2,14 @@ import stringifyObject from 'stringify-object';
 import { ScryptedInterface, ScryptedInterfaceDescriptor } from "./types.input";
 import path from 'path';
 import fs from "fs";
+import packageJson from "../package.json"
```

Comment:
We can import `package.json` directly instead of using `JSON.parse(fs.readFileSync(`